_data/simple-icons.json

```
@@ -3078,6 +3078,12 @@
                 "type": "MIT"
             }
         },
+        {
+            "title": "Denon",
+            "hex": "FFFFFE",
```

```suggestion
            "hex": "0B131A",
```
I'd go with this personally, most of their products are black, and they use this in the site header and various social channels.